brain-marks/Localization/hi.lproj/Localizable.strings

```diff
@@ -0,0 +1,55 @@
+/*
+  Localizable.strings
+  brain-marks
+
+  Created by AC Richter on 13.12.21.
```

Comment:
Please change this comment, you can add your own name and the date you created this file (and several others)